test/BUILD.gn

```diff
@@ -575,6 +575,7 @@ test("brave_unit_tests") {
 
   if (enable_brave_wayback_machine) {
     sources += [
+      "//brave/components/brave_wayback_machine/brave_wayback_machine_policy_unittest.cc",
```

Comment:
AFAIK, we don't want to add indivisual test file to `test/BUILD.gn` directly.
It's recommended to have separated unittest target like`//brave/comonents/brave_wayback_machine:unittest` and adding to `//brave/components:brave_components_unittests` target instead of here.

Reply:
Fixed in 772b4e84c682a5fdf809622f8805c8005eff2196

Reply:
Fixed in 
855d1b0d87eee973683d63c02a2db45e25bddaeb
For this unit test and the others that existed before